LoopKit/DeviceManager/PumpManager.swift

```diff
@@ -50,6 +50,15 @@ public protocol PumpManagerDelegate: DeviceManagerDelegate, PumpManagerStatusObs
 
 
 public protocol PumpManager: DeviceManager {
+    /// The maximum number of scheduled basal rates in a single day supported by the pump. Used during onboarding by therapy settings.
```

Comment:
[nit] would it be possible to put these three things into a `struct` instead of having three `get`ters?  I'm not sure how it is intended to be used.  Just a suggestion...

Reply:
This code mirrors the similar non-static functions below (which have existed for a long time). I agree it would make sense to use a struct, but I wanted to be consistent.